crates/ibc-query/src/core/channel/query.rs

```diff
@@ -469,39 +469,10 @@ where
     })
 }
 
-/// Queries for the next sequence send associated with a channel
-pub fn query_next_sequence_send<I>(
-    ibc_ctx: &I,
-    request: &QueryNextSequenceSendRequest,
-) -> Result<QueryNextSequenceSendResponse, QueryError>
-where
-    I: ValidationContext + ProvableContext,
-{
-    let channel_id = ChannelId::from_str(request.channel_id.as_str())?;
-
-    let port_id = PortId::from_str(request.port_id.as_str())?;
-
-    let next_seq_send_path = SeqSendPath::new(&port_id, &channel_id);
-
-    let next_sequence_send = ibc_ctx.get_next_sequence_send(&next_seq_send_path)?;
-
-    let current_height = ibc_ctx.host_height()?;
-
-    let proof = ibc_ctx
-        .get_proof(current_height, &Path::SeqSend(next_seq_send_path))
-        .ok_or(QueryError::ProofNotFound {
-            description: format!(
-                "Next sequence send proof not found for channel {}",
-                channel_id
-            ),
-        })?;
-
-    Ok(QueryNextSequenceSendResponse {
-        next_sequence_send: next_sequence_send.into(),
-        proof,
-        proof_height: Some(current_height.into()),
-    })
-}
+// NOTE: The `query_next_sequence_send` method is currently not present in the
+// generated protobuf from `ibc-go` v0.7.3. However, this method has been
+// reintroduced through [this PR](https://github.com/cosmos/ibc-go/pull/3417) in
+// ibc-go. Once included in a new release, we can implement this method call.
```

Comment:
I think this comment would look strange without the method here. Maybe we should keep the function - as it is or commented out. Does it make sense?

Reply:
Keeping the function as is does not compile since relevant types are missing too. 
I'd rather avoid leaving the commented-out code anywhere, particularly as it would remain in our releases. 
Mainly added this comment for maintainers to address later.

Reply:
[d8a91b5](https://github.com/cosmos/ibc-rs/pull/895/commits/d8a91b52b35b89c5a89ffa9f4cee397c6043a6a4)